assets/queries/cloudFormation/sns_topic_subscription_not_setted/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "SNS_Topic_Subscription_Not_Setted",
+  "queryName": "SNS Topic Subscription Not Setted",
```

Comment:
Change **directory name** and **id** to: `sns_topic_without_subscriber`
Change **query name** to: `SNS Topic Without Subscriber`